templates/guides/body/admonition.html.twig

```diff
@@ -0,0 +1,30 @@
+{% if name == 'important' %}
+{% endif %}
+{% if name == 'note' %}
+    {% set admonitionClass='note' %}
+    {% set textClass='text-primary' %}
+    {% set icon='fa-sticky-note' %}
+{% endif %}
+{% if name == 'warning' or name == 'caution' %}
+    {% set admonitionClass='warning' %}
+    {% set textClass='text-primary' %}
+    {% set icon='fa-exclamation-circle' %}
+{% endif %}
+{% if name == 'tip' or name == 'hint' %}
+{% endif %}
+{% if name == 'seealso' %}
+{% endif %}
```

Comment:
Are these 2 blocks missing some calls to `set`?

Reply:
Could be, that these are needed. I need to check if the documentation contains those types.